canonical-websites/www.ubuntu.com
# Created Juju Takeover initial draft

## Done

Created the juju takeover page 
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/statistics#desktop-specs](http://0.0.0.0:8001/statistics#desktop-specs)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check it matches https://docs.google.com/document/d/1e2qjJUrcgZVsvYeR9GgCu6nHUWFMkcDEstAKTqE4k5M/edit#heading=h.meo0azuz09li


## Issue / Card

Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4145
